R/num-ansi-colors.R

```diff
@@ -244,7 +244,7 @@ emacs_version <- function() {
 
 win10_build <- function() {
   os <- utils::sessionInfo()$running
-  if (!grepl("^Windows 10 ", os)) return(0L)
+  if (isFALSE(grepl("^Windows 10 ", os))) return(0L)
```

Comment:
Unfortunately `isFALSE()` needs R-3.5, so we can't use that, or we need to supply a wrapper.

I suggest we do this instead:
```r
os <- utils::sessionInfo()$running %||% ""
```

Reply:
Sorry, I didn't know about this limitation. I changed it according to your suggestion.